src/test/scala/io.circe.config/CirceConfigSpec.scala

```diff
@@ -44,57 +45,77 @@ class CirceConfigSpec extends AnyFlatSpec with Matchers {
     assert(config.l.unwrapped == "localhost")
   }
 
-  "parser" should "parse and decode config from string" in new ParserTests {
-    def parse = parser.parse(AppConfigString)
-    def decode = parser.decode[TestConfig](AppConfigString)
+  def parserAssertions(parse: Either[ParsingFailure, Json], decode: Either[Error, TestConfig]): Unit = {
+    assert(parse.isRight)
+
+    val Right(config) = decode
+
+    assert(config == DecodedTestConfig)
```

Comment:
For these assertions checking for equality, you can use `assertEquals` https://scalameta.org/munit/docs/assertions.html#assertequals